agrirouter/messaging/services/messaging.py

```diff
@@ -276,3 +276,39 @@ def encode(parameters: SubscriptionParameters) -> EncodedMessage:
         )
 
         return encoded_message
+
+
+class SendMessageService(AbstractService):
+    """
+    Service for sending messages to the agrirouter
+    """
+
+    @staticmethod
+    def encode(parameters: SendMessageParameters) -> EncodedMessage:
+        """
+        Encode the parameters into a message.
+        parameters: Parameters for the message service.
+        """
+        message_header_parameters = MessageHeaderParameters(
+            technical_message_type=parameters.get_technical_message_type(),
+            mode=RequestEnvelope.Mode.Value("DIRECT"),
```

Comment:
This has to be part of the parameters, since it is not the only way to send messages.

Reply:
So the Parameters class should be used? For all the other services MessageHeaderParameters class is used.